addons/portal_rating/static/src/js/portal_composer.js

```diff
@@ -61,19 +61,38 @@ PortalComposer.include({
         var self = this;
         return this._super.apply(this, arguments).then(function () {
             // rating stars
-            self.$input = self.$('input[name="rating_value"]');
+            self.$inputRating = self.$('input[name="rating_value"]');
             self.$star_list = self.$('.stars').find('i');
 
             // set the default value to trigger the display of star widget and update the hidden input value.
             self.set("star_value", self.options.default_rating_value); 
-            self.$input.val(self.options.default_rating_value * STAR_RATING_RATIO);
+            self.$inputRating.val(self.options.default_rating_value * STAR_RATING_RATIO);
         });
     },
 
     //--------------------------------------------------------------------------
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     * @private
+     */
+    _onSubmitButtonClick: function (ev) {
+        //close rating bootstrap modal
+        this.$el.closest('#ratingpopupcomposer').modal('hide');
```

Comment:
The modal shouldn't be hidden if the submit was unsuccessful. Show a warning that the comment is required and keep the modal open.